api/database/migrations/2025_05_15_204145_remove_team_department_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('team_department', function (Blueprint $table) {
+            $table->dropForeign(['team_id']);
+            $table->dropForeign(['department_id']);
+            $table->dropColumn('team_id');
+            $table->dropColumn('department_id');
+        });
+        Schema::dropIfExists('team_department');
+    }
+
+    /**
+     * Reverse the migrations.
+     */
+    public function down(): void
```

Comment:
I'm not confident that this is correct.

Reply:
Yeah, you'll need to create the pivot

See this for reference

![image](https://github.com/user-attachments/assets/2718c4c7-4ed2-435a-b52b-b3d85156040e)